networkx/algorithms/centrality/domirank.py

```diff
@@ -0,0 +1,305 @@
+"""DomiRank Centrality: By Marcus Engsig (@mengsig)"""
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+
+__all__ = ["domirank"]
+
+
+@not_implemented_for("multigraph")
+@nx._dispatchable(edge_attrs="weight")
+def domirank(
```

Comment:
Follow the naming convention of the other centrality metrics.

```suggestion
def domirank_centrality(
```